### DISCUSSION TITLE
各位覺得考古題該傳嗎？

### DISCUSSION THREAD

**Discussion by JaYanlloz** [1101-GS4719-C/Python-Community-GS4719-C#2188] [General]
最近Dcard中央版上有蠻多文章在討論考古題的，期中考將至，相信大家也開始遇到很多有關考古的問題了，之前有遇過因為考古題氾濫而被教授大幅提升難度的考古題之亂，現在自己有要考的科目被要考古都怕怕的，到底該不該傳阿。
總覺得不傳蠻傷感情的說，但傳了一個就得傳無數個...

**Comment by Mminniehuang** [comment]
首先我覺得先不討論教授並沒有同意外傳的那種考古，這可能涉及到尊不尊重的問題。
其他我覺得並沒有什麼不好，但我畢竟沒有經歷過大幅提升難度的狀況，說不定我遇到了會有完全不同的感受。
之前有一門課的教授考前給我們考古題，但真正考試的題目敘述不會完全一樣，要真的搞懂而不是死背，但又給我們了準備的方向，包括題型、考試著重部分等等，雖然準備的時候還是有點心累和焦慮，但真的能更有效率學到該學的，我想這樣也才是考古題的意義所在。

**Comment by wcyeee** [comment]
我覺得考古題有點是提早讓我們練習以後職場到底要不要分享「秘訣」或是互相幫助。 相信每次有人來要考古題的狀況應該都不太一樣，有時候似乎可以拒絕，有時候又拒絕不下去。我個人覺得，如果這個人看起來是想要透過考古題變得更好，像是透過練習提高自己對課程的了解度的話我是會給他；但如果這個人就是看起來是想走捷徑的人，我如果可以拒絕我就會拒絕！（如果真的因為人情壓力沒辦法拒絕，可以想想看他拿到之後只會對成績平均有影響 雖然可能自己就不會分數那麼高 但那個人什麼都沒學到 這是他的超級大損失 他以後會有代價的！）
感覺就有點像是職場如果是一個超級認真的同事來問你秘訣 我感覺就會很快樂的給他跟他一起討論 如果是走捷徑的我就會說不知道：）

**Comment by Tchang323** [comment]
在考前刷考古能有一定程度對考試內容有了解，實際考試可以更流暢的作答。
我認為對於大多數人而言，一定會希望考前能有考古題複習。然而，這可能也會影響大家學習的成效，尤其針對部分科目的考試跟歷年考古很相似時，學生可能會選擇去背答案，而不是真正去理解原理。

**Comment by Jinyuu17** [comment]
這完全取決於你自己的意願。如果你覺得分享考古題能夠幫助學弟妹提高他們的學習成果，而且你認為這是一種支持學習社群的方式，那麼分享是可以的。如果你覺得這樣做會讓他們失去自主學習的機會，或讓學習過程變得過於簡單，那麼保持不分享也是一種選擇。

**Comment by lai1008** [comment]
在考前刷考古能有一定程度對考試內容有了解，實際考試可以更流暢的作答。
我認為對於大多數人而言，一定會希望考前能有考古題複習。然而，考古題也會影響大家學習的成效，當部分科目的考試跟歷年考古很相似時，學生可能會選擇去背答案。

**Comment by ed112601513** [comment]
我認為刷考古題能讓自己更清楚出題方向和知識重點，以前國中會考、高中學測和分科，我相信大部分人一定也都靠寫考古題來確定自己的學習是否到位，所以我認為只要找的到考古題就可以寫看看。而大學有些教授的考題每年都差不多，所以不開放傳閱考古題，但依舊有人外流，對於這個問題我的觀點是只要能從考古題學到知識並且能實用，那又有何不可呢?

**Comment by 112121py** [comment]
我覺得考古題可以讓我們比較知道題目的出題方式的內容，也是一個讓我們練習自己是否準備好的一個管道，但考古題就是會有萬一考題出得幾乎一樣，那有些人就會選擇直接背答案，懶得去理解，當然除了沒有好好理解之外，如郭突然出了不一樣的題目，那就虧大了。所以考古題我覺得沒有不好，只是你要怎麼利用他，來讓你有夠好的學習效率

**Comment by christaaah** [comment]
就我而言，考古題可以讓我更知道教授繪出的題型是什麼，對於準備也更有方向，這些考古題，就像以前學測時會寫的歷屆考題一樣。如果教授每一年出的題目是不一樣的，我認為傳考古題沒有什麼太大的影響，但如果出的考卷題目是一樣的，爭議就會比較大。

**Comment by yue083139** [comment]
以我的經驗來說，考古題真的是一個很方便的東西，像之前有段時間段考全部都擠在一起，沒辦法仔細地讀筆記或課本時，刷考古題就變成了最有效率的準備方法，畢竟會出在考古題的題目都是學習的重點，而就算準備時間充裕，考古題也可以充當驗證我們學習成效的工具。

**Comment by forestdew507** [comment]
從學生的視角來看，我認為有沒有考古並沒有絕對的好壞，好壞取決於學生個案的使用方法/使用心態以及面對的科目差異。畢竟有些學生是透過題目來完善自己的認知。

**Comment by 113607514** [comment]
我認為考古題還是應該傳的，因為它不僅能幫助大家更好地準備考試，也能提升整體的學習效果。傳考古題讓我們能知道考試的方向，了解題型和重點，這對於複習來說非常關鍵。與其自己一個人緊張，不如大家一起分享、互相幫助，把學習變成更有支持感的過程。

**Comment by gigitilto** [comment]
我認為這不是好事，但可能也還好，因為在我看來，考古題就只是提供老師的出題習慣，若給了考古題，可能會導致某些學生只學會考古題的內容，一但老師改變出題方向，會導致學生成績不及格，也學不到東西。可是提供考古題也有好處，可以讓有在認真讀書的同學更有信心，在做題的時候更有把握，即使考試沒有出出來，也會因為慶幸自己有讀書而不會因為心情影響考試。

**Comment by ShuliANN** [comment]
我認同上述同學所說的，考古題這件事的好壞取決於學生個案的使用方法/使用心態。像我自己在使用考古題時，不但可以幫助我複習，更可以有效地抓住重點，不過當然如果有人根本沒心學習，只利用考前刷考古而得高分，那麼沒有學到東西，也是自己的損失，對別人不會有影響！

**Comment by Taoyi-Chiang** [comment]
我覺得如果看得出來老師會題高難度，那就應該要偷偷做吧？
只是好像很多人都無法理解有些事情「只能做不能說」的道理wwwwww

**Comment by Boying12** [comment]
我完全理解你的看法，考古題的分享確實是一把雙刃劍。雖然考古題可以幫助同學們更好地準備考試，但在某些情況下，這樣的分享可能會引發不必要的麻煩。尤其是當遇到一些人想利用考古題來進行資產交換，卻不願意提供相應的回報時，這種行為不僅讓人感到失望，還會影響到彼此的信任。
在學習的過程中，我們應該互相支持，分享資源是為了共同進步，而不是成為某些人私利的工具。如果考古題的分享變成了一種內捲的競爭，甚至出現不誠實的行為，那麼這樣的交流就失去了原本的意義，反而讓人感到厭煩。
因此，我認為在分享考古題時，應該謹慎選擇對象，避免與那些只圖利而不肯回報的人交涉。最好的方式是尋找志同道合的同學，建立真誠的合作關係，讓學習變得更加愉快和有意義。希望大家能在這個過程中共同成長，而不是陷入不必要的糾紛中！
